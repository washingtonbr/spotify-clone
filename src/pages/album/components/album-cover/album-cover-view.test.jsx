import React from 'react';
import renderer from 'react-test-renderer';
import AlbumCoverView from './album-cover-view';

describe('AlbumCoverView', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(
        <AlbumCoverView
          cover="https://i.scdn.co/image/ab67616d0000b2732f44aec83b20e40f3baef73c"
          name="Music To Be Murdered By"
          artists={['Eminem']}
        />,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
