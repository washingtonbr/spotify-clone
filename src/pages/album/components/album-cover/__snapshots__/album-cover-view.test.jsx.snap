// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumCoverView renders correctly 1`] = `
<div
  className="sc-AxjAm flemmw"
>
  <img
    alt="Music To Be Murdered By"
    className="sc-AxirZ izKjuu"
    src="https://i.scdn.co/image/ab67616d0000b2732f44aec83b20e40f3baef73c"
  />
  <h2
    className="sc-AxiKw bKnEfS"
  >
    Music To Be Murdered By
  </h2>
  <h3
    className="sc-AxhCb csvEgr"
  >
    Eminem
  </h3>
</div>
`;
